pydifact/segmentcollection.py

```diff
@@ -364,7 +364,7 @@ def __init__(
         sender: Element,
         recipient: Element,
         control_reference: Element,
-        syntax_identifier: Element,
+        syntax_identifier: tuple[str, int],
```

Comment:
I'd like to move here to an even more specific Element, but tuple[str, int] is ok for me for the moment.